src/routes/hooks/hooks-cache.spec.ts

```diff
@@ -1128,11 +927,42 @@ describe.skip('Post Hook Events for Cache (Unit)', () => {
       faker.number.int({ min: 1 }),
     );
 
-    await request(app.getHttpServer())
-      .post(`/hooks/events`)
-      .set('Authorization', `Basic ${authToken}`)
-      .send(data)
-      .expect(202);
+    const cb = queuesApiService.subscribe.mock.calls[0][1];
+    await cb({ content: Buffer.from(JSON.stringify(data)) } as ConsumeMessage);
+
+    await expect(fakeCacheService.hGet(cacheDir)).resolves.toBeUndefined();
+  });
+
+  it.each(
+    [
+      newDelegateEventBuilder().build(),
+      updatedDelegateEventBuilder().build(),
+      deletedDelegateEventBuilder().build(),
+    ].map((event) => [event.type, event]),
+  )('%s clears delegates', async (_, event) => {
```

Comment:
These were added as they were missing - initially marked with a TODO at the top.